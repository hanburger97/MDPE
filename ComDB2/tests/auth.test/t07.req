#this should fail
select 1
put password 'password3' for 'user3'
#only op user can add new user
set user user1
set password password1
put password 'password3' for 'user3'
