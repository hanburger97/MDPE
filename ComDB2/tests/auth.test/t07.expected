(1=1)
[put password 'password3' for 'user3'] failed with rc -3 User does not have OP credentials
